cli/cmd/create.go

```diff
@@ -52,7 +59,6 @@ Built-in templates:
 	}
 
 	createCmd.Flags().StringVarP(&appName, "name", "n", "", "Application name")
-	createCmd.MarkFlagRequired("name")
```

Comment:
Removed, because it forces flag's completion. Instead of it, a check for the absence of a flag has been added.